Include limits.h in growisofs.c and transport.hxx to make
them able to use CDSL_CURRENT and avoid compilation issues.

diff -Nau dvd+rw-tools-7.0/growisofs.c dvd+rw-tools-7.0.new/growisofs.c
--- dvd+rw-tools-7.0/growisofs.c	2008-03-29 11:19:22.000000000 +0100
+++ dvd+rw-tools-7.0.new/growisofs.c	2008-03-29 11:20:33.000000000 +0100
@@ -419,6 +419,7 @@
 #include <sys/stat.h>
 #include <assert.h>
 #include "mp.h"
+#include <limits.h>
 
 #if defined(__unix) || defined(__unix__)
 # include <unistd.h>
diff -Nau dvd+rw-tools-7.0/transport.hxx dvd+rw-tools-7.0.new/transport.hxx
--- dvd+rw-tools-7.0/transport.hxx	2008-03-29 11:19:22.000000000 +0100
+++ dvd+rw-tools-7.0.new/transport.hxx	2008-03-29 11:24:28.000000000 +0100
@@ -16,6 +16,7 @@
 #include <fcntl.h>
 #include <poll.h>
 #include <sys/time.h>
+#include <limits.h>
 
 inline long getmsecs()
 { struct timeval tv;
